client/src/templates/Challenges/classic/lower-jaw.css

```diff
@@ -0,0 +1,10 @@
+.lower-utility-bar .btn {
+  position: relative;
+}
+
+.lower-utility-bar .btn span {
+  position: absolute;
+  top: 50%;
+  right: 12%;
+  transform: translateY(-50%);
+}
```

Comment:
I would urge you to reconsider absolute positioning here. I think it might be much easier to use flex on the button itself to position the text and icon. Also, this will fix your problems with rtl direction.